homeassistant/components/lovelace/__init__.py

```diff
@@ -18,10 +18,24 @@
 LOVELACE_CONFIG_FILE = 'ui-lovelace.yaml'
 JSON_TYPE = Union[List, Dict, str]  # pylint: disable=invalid-name
 
+FORMAT_YAML = 'yaml'
+FORMAT_JSON = 'json'
+
 OLD_WS_TYPE_GET_LOVELACE_UI = 'frontend/lovelace_config'
 WS_TYPE_GET_LOVELACE_UI = 'lovelace/config'
+
 WS_TYPE_GET_CARD = 'lovelace/config/card/get'
 WS_TYPE_SET_CARD = 'lovelace/config/card/set'
+WS_TYPE_ADD_CARD = 'lovelace/config/card/add'
```

Comment:
Our lived would be a lot easier if you either have a single commit per WS command, so it's easy to review. Or if you do a PR per command.

Reply:
Will do, tomorrow night I'll have time to work on it.